From a7eb3cb599fbb1d66c6980d53550822cbbfd1520 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Wed, 14 Aug 2019 11:25:49 +0200
Subject: Add missing headers with DISABLE_MODEMMANAGER_SUPPORT=ON

Reviewers: jgrulich

Reviewed By: jgrulich

Subscribers: plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D23148
---
 libs/declarative/networkstatus.cpp | 2 ++
 libs/handler.cpp                   | 1 +
 libs/handler.h                     | 1 +
 3 files changed, 4 insertions(+)

diff --git a/libs/declarative/networkstatus.cpp b/libs/declarative/networkstatus.cpp
index b9db376..e94b7ea 100644
--- a/libs/declarative/networkstatus.cpp
+++ b/libs/declarative/networkstatus.cpp
@@ -20,7 +20,9 @@
 
 #include "networkstatus.h"
 #include "uiutils.h"
+
 #include <QDBusConnection>
+#include <QDBusConnectionInterface>
 
 #include <NetworkManagerQt/ActiveConnection>
 #include <NetworkManagerQt/Connection>
diff --git a/libs/handler.cpp b/libs/handler.cpp
index 40b4b43..0095aa7 100644
--- a/libs/handler.cpp
+++ b/libs/handler.cpp
@@ -46,6 +46,7 @@
 #endif
 
 #include <QDBusError>
+#include <QDBusMetaType>
 #include <QDBusPendingReply>
 #include <QIcon>
 
diff --git a/libs/handler.h b/libs/handler.h
index f8b7941..4c72699 100644
--- a/libs/handler.h
+++ b/libs/handler.h
@@ -22,6 +22,7 @@
 #define PLASMA_NM_HANDLER_H
 
 #include <QDBusInterface>
+#include <QTimer>
 
 #include <NetworkManagerQt/Connection>
 #include <NetworkManagerQt/Settings>
-- 
cgit v1.1


